src/surveys/components/RangeResponse.tsx

```diff
@@ -87,10 +89,9 @@ function RangeResponse({ question, onAnswered, theme }: RangeResponseProps) {
   return (
     <div className='space-y-3 w-[504px] overflow-x-auto'>
       <Header
-        title={question.label}
-        description={question.description}
+        title={label}
+        description={description}
         color={theme?.question}
-        centered={question.type === 'rating'}
```

Comment:
@jdamilola The header doesn't need to be centered for rating questions anymore right?

Reply:
@AfeezGL Yes, All headers and descriptions should follow the same pattern